hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/TssBaseServiceImpl.java

```diff
@@ -114,10 +115,14 @@ public void bootstrapLedgerId(
     }
 
     @Override
-    public void setCandidateRoster(@NonNull final Roster roster, @NonNull final HandleContext context) {
-        requireNonNull(roster);
-        // (TSS-FUTURE) https://github.com/hashgraph/hedera-services/issues/14748
-        tssSubmissions.submitTssMessage(TssMessageTransactionBody.DEFAULT, context);
+    public void setCandidateRoster(@NonNull final Roster newCandidate, @NonNull final HandleContext context) {
+        requireNonNull(newCandidate);
+
+        final var rosterStore = context.storeFactory().writableStore(WritableRosterStore.class);
```

Comment:
While testing, I found that this place will be called by `NodeStakeUpdate` and the `writableStoreFactory` here will be from `TokenService`. So here, we cant use the same context to construct rosterStore. 
We can move thos logic to `nodeStakeUpdates` and do like this. 
```
final var rosterStore = writableStoreFactory.getStore(WritableRosterStore.class);
        if (!Objects.equals(newCandidateRoster, rosterStore.getCandidateRoster())
                && !Objects.equals(newCandidateRoster, rosterStore.getActiveRoster())) {
            rosterStore.putCandidateRoster(newCandidateRoster);
        }
```

Reply:
Done